_tools/libddwaf-updater/update.go

```diff
@@ -34,24 +35,35 @@ const (
 	goVersionUnsupported = "go1.23"
 )
 
+var (
+	forceFlag   *bool
+	versionFlag *string
```

Comment:
I'd rather these weren't pointers...

```suggestion
	forceFlag   bool
	versionFlag string
```